packages/protocol/src/interfaces/dforce/IController.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.15;
+
+interface IController {
```

Comment:
You're right. Will change it as well

Reply:
The IController was deleted because it was redundant. 
The other three interfaces have the mint, redeemUnderlying and redeem methods which have one/two arguments. In the compound interfaces it only has none/one. If I change them to Compound and delete the argument, it throws a delegate call fail because dforce doesnt have those functions with less arguments.
There's also an option of still eliminating those interfaces and adding those functions to compound interfaces but doesn't feel right.
Let me know your thoughts on this @DaigaroCota .